NEWS

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+#
+
+## Current
```

Comment:
should we set a release version + date instead? Like we do in [UCX repo](https://github.com/openucx/ucx/releases)

Reply:
added specific version and date